metaflow/cli.py

```diff
@@ -21,11 +23,11 @@
     DEFAULT_METADATA,
     DEFAULT_MONITOR,
     DEFAULT_PACKAGE_SUFFIXES,
+    FLOW_VALIDATORS  # could be 'FLOW_LINTERS', too- they help choose linter type
```

Comment:
@savingoyal this is config of the Validators a.k.a Linters here- we almost dont need pylint once we vet ruff. But still, FWIW, I have a config(and possibly capability(if not support, hereafter) for both).